etest/docker/container.py

```diff
@@ -48,13 +50,13 @@ def logs(*args, **kwargs):
     return common.API_CLIENT.logs(*args, **kwargs)
 
 
-def remove(container, container_name, *args, **kwargs):
+def remove(container: Container, container_name: str, *args, **kwargs):
```

Comment:
Does container contain the container_name?  If so, we can reduce the arity here.

Reply:
Actually, it does. I didnt think about that.